test/JDBC/input/ISC-Views.sql

```diff
@@ -77,6 +77,12 @@ go
 create view vcnums as select * from nums with check option;
 go
 
+create view iscv as select * from information_schema.views;
+go
+
+create view sbv with schemabinding as select 1;
```

Comment:
You don't need this test because `vu` tests will be executed in both normal and upgrade scenario.